src/odemis/acq/fastem.py

```diff
@@ -120,16 +125,33 @@ def _calculate_field_indices(self):
         :return: (list of nested tuples (col, row)) The column and row field indices of the field images in the order
                  they should be acquired. The tuples are re-ordered so that the single field images resembling the
                  ROA are acquired first rows then columns.
+
+        Note: The number of fields with overlap is calculated in the following way:
+        An ROA with overlap can be visualized as follows:
+        |------|--⁞----|--⁞----|--⁞
+        If you have 3 fields of 6400 pixels wide with 400 pixels overlap, the resulting megafield is
+        6000 * 3 + 400 pixels in width and the width is abs(r - l). The number of fields that fit in a
+        certain width can be derived from this:
+        abs(r - l) = 6000 * 3 + 400
+        (abs(r-l) - 400) / 6000 = 3 fields
+        with:
+        400 = 6400 * overlap = field_size * overlap
+        6000 = 6400 * (1 - overlap) = field_size * (1 - overlap)
+        => n_fields = (abs(r - l) - field_size * overlap) / (field_size * (1 - overlap))
```

Comment:
Two notes.
Firstly, if the user requests a very tiny region, less than the overlap (eg, abs(r-l) == 300px), you end up with negative numbers, which eventually get rounded up to 0. It should probably handle such case. Maybe just use a "if width < field_size, then n == 1".

Secondly, be aware that in the tiledacq code we use a different formula. We use the simpler n_fields = abs(r - l) / (field_size * (1 - overlap)) . Basically, it considers that the first field is also smaller. The reason for this (in addition to simplifying a lot of code) is that the overlap is there to compensate for the stage (or other hw) inaccuracy. So you cannot really consider that the first field is perfectly overlapping the requested area. Therefore it's safer to also consider the first field smaller.